src/EVEMon.Common/Helpers/CharacterScratchpad.cs

```diff
@@ -44,6 +44,34 @@ public CharacterScratchpad(BaseCharacter character)
             Reset();
         }
 
+        /// <summary>
+        /// Check for alpha clone state.
+        /// </summary>
+        public bool IsAlpha
```

Comment:
This chain of `instanceof` checks will be difficult to maintain. Consider instead making `IsAlpha` a virtual property on `BaseCharacter` and returning `false`, then overriding as you currently do in `Character`. `CharacterScratchpad` can then unconditionally delegate to `m_character.IsAlpha`